crates/ide/src/move_item.rs

```diff
@@ -121,6 +141,17 @@ fn replace_nodes(first: &SyntaxNode, second: &SyntaxNode) -> TextEdit {
     edit.finish()
 }
 
+fn pick_best(tokens: TokenAtOffset<SyntaxToken>) -> Option<SyntaxToken> {
+    return tokens.max_by_key(priority);
+    fn priority(n: &SyntaxToken) -> usize {
```

Comment:
```suggestion
    return tokens.max_by_key(priority);
    
    fn priority(n: &SyntaxToken) -> usize {
```

Reply:
I'll apply fix to https://github.com/rust-analyzer/rust-analyzer/blob/563464bfaeec9e39de52cd12c435c299fe544b71/crates/ide/src/hover.rs#L471-L475 too